recipes/rocksdb/all/conanfile.py

```diff
@@ -0,0 +1,157 @@
+        self._cmake.definitions["WITH_GFLAGS"] = self.options.with_gflags
+        self._cmake.definitions["WITH_SNAPPY"] = self.options.with_snappy
+        self._cmake.definitions["WITH_LZ4"] = self.options.with_lz4
+        self._cmake.definitions["WITH_ZLIB"] = self.options.with_zlib
+        self._cmake.definitions["WITH_ZSTD"] = self.options.with_zstd
+        self._cmake.definitions["WITH_TBB"] = self.options.with_tbb
+        self._cmake.definitions["ROCKSDB_BUILD_SHARED"] = self.options.shared
+        self._cmake.definitions["ROCKSDB_LIBRARY_EXPORTS"] = self.settings.os == "Windows" and self.options.shared
+        self._cmake.definitions["ROCKSDB_DLL" ] = self.settings.os == "Windows" and self.options.shared
+
+        # not available yet in CCI
+        self._cmake.definitions["WITH_JEMALLOC"] = False
+        self._cmake.definitions["WITH_NUMA"] = False
+
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def requirements(self):
+        if self.options.with_gflags:
+            self.requires("gflags/2.2.2")
+        if self.options.with_snappy:
+            self.requires("snappy/1.1.7")
+        if self.options.with_lz4:
+            self.requires("lz4/1.9.2")
+        if self.options.with_zlib:
+            self.requires("zlib/1.2.11")
+        if self.options.with_zstd:
+            self.requires("zstd/1.3.8")
+        if self.options.with_tbb:
+            self.requires("tbb/2019_u9")
+
+    def _remove_static_libraries(self):
+        for static_lib_name in ["lib*.a", "{}.lib".format(self.name)]:
+            for file in glob.glob(os.path.join(self.package_folder, "lib", static_lib_name)):
+                os.remove(file)
+
+
+    def _remove_cpp_headers(self):
+        for path in glob.glob(os.path.join(self.package_folder, "include", "rocksdb", "*")):
+            if path != os.path.join(self.package_folder, "include", "rocksdb", "c.h"):
+                if os.path.isfile(path): 
+                    os.remove(path)
+                else:
+                    shutil.rmtree(path)
+
+    def package(self):
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
+        self.copy("LICENSE*", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        if self.options.shared:
+            self._remove_static_libraries()
+            self._remove_cpp_headers() # Force stable ABI for shared libraries
+
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "RocksDB"
+        self.cpp_info.names["cmake_find_package_multi"] = "RocksDB"
+        self.cpp_info.libs = tools.collect_libs(self)
+        if self.settings.os == "Windows":
+            self.cpp_info.system_libs = ["Shlwapi.lib", "Rpcrt4.lib"]
```

Comment:
I think the ".lib" extensions are not needed
```suggestion
            self.cpp_info.system_libs = ["Shlwapi", "Rpcrt4"]
```